docs/web/server_config.md

```diff
@@ -17,6 +17,12 @@ Table of Contents
     * [Size of the compilation database](#size-of-the-compilation-database)
 * [Authentication](#authentication)
 
+## Number of worker processes
+The `worker_processes` section of the config file controls how many processes
+will be started on the server to process API requests.
+
```

Comment:
Please add a note that the server needs to be restarted if the value is changed in the config file.